content/en/observability_pipelines/troubleshooting.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Troubleshooting
+kind: documentation
+disable_toc: false
+---
+
+## Overview
+If you experience unexpected behavior with Datadog Observability Pipelines (OP), there are a few common issues you can investigate, and this guide may help resolve issues quickly. If you continue to have trouble, reach out to Datadog support for further assistance.
+
+## Investigate diagnostic logs
+
+The Observability Pipelines Worker emits internal logs about its own health. In the Observability Pipelines UI, you can investigate any internal error logs that the Worker process emits for all of its individual components. To see these diagnostic logs:
+
+1. Navigate to [Observability Pipelines][1].
+1. Click into the pipeline you want to investigate.
+1. Click on a component to see the component's side panel.
+1. Click the **Diagnostic Logs** tab to view error logs that the Worker is emitting. Click on a log record to investigate it in the Log Explorer. If there are no logs listed, then the component is not emitting any error logs.
+
+### Get more detailed logs
+
+If you need more details about the internal logs the OP Worker collects, you can increase the level of the logs using the `VECTOR_LOG` environment variable. By default, this is set to `INFO`, which means that `INFO`, `WARNING`, and `ERROR` messages show in the console.
```

Comment:
This might just be my lack of understanding with the OP Worker logs, but where would a customer configure the `VECTOR_LOG` envar? Is it in the `datadog.yaml` ? If so I would just maybe add this here.

Reply:
Good question...@silversupreme could you let us know?

Reply:
Found out it is different depending on the installation... Need to add sections for Helm/CloudFormation/Linux/Docker. Have a separate to card to add this info so that it's not blocking merge. Created [DOCS-6451](https://datadoghq.atlassian.net/browse/DOCS-6451)